astrovascpy/utils.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 import pandas as pd
 import psutil
-import scipy as sp
```

Comment:
According to [Scipy documentation](guidelines-for-importing-functions-from-scipy), the proper way to load symbols is to load the `sparse` Scipy submodule:

```python
from scipy import sparse
```

There is no reason to import the scipy module itself as it only provides Numpy functions (for backward compatibilities).

Reply:
Sounds good!